hushline/model/user.py

```diff
@@ -179,6 +179,15 @@ def max_aliases(self) -> int:
         current_app.logger.warning(err_msg)
         return self._PREMIUM_ALIAS_COUNT
 
+    @property
```

Comment:
very much like that this is its own property. more understandable, easier to refactor :D